lib/parity/environment.rb

```diff
@@ -128,7 +138,7 @@ def basename
     end
 
     def run_migration?
-      rails_app? && pending_migrations?
+      pending_migrations? && rails_app?
```

Comment:
No. The `rails_app?` step takes longer, so I was putting it off until it was necessary, but we should constrain the scope here, I'll undo this change.